docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,467 @@
+# Component Port
+
+Tizen components of the component-based applications can communicate with each other using component ports. Components can send and receive serializable objects through component port communication.
+
+The main feature of the `Tizen.Applications.ComponentBased.ComponentPort` class include:
+- Managing component port
+
+  You can set up the component ports to [send and receive requests](#port) between components of component-based application with the `Tizen.Applications.ComponentBased.ComponentPort` class.
+
+- Using component task
+
+  You can set up the component tasks to [run tasks](#task) and wait for events from other components with the `Tizen.Applications.ComponentBased.ComponentTask` class.
+
+## Prerequisites
+
+To enable your component to use the component port functionality:
+
```

Comment:
Just a doubt, 
Dont we need privileges ?

To use classes and methods, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:

Reply:
Privileges are not needed.
When creating the port, the own socket of the application is created.
The application can add privileges using AddPrivilege() for access control.

Reply:
sure, thanks.